src/tag/tag.js

```diff
@@ -53,14 +53,15 @@ class Tag extends React.Component<PropsT, {}> {
       $disabled: disabled,
       $color: color,
     };
+    const events = disabled
+      ? {}
+      : {
+          onClick: e => onActionClick(e, children),
+        };
     return (
       <Root {...sharedProps} {...rootProps}>
         {children}
-        <Action
-          {...sharedProps}
-          {...actionProps}
-          onClick={e => onActionClick(e, children)}
-        >
+        <Action {...sharedProps} {...actionProps} {...events}>
```

Comment:
The props overrides seems to be missing here

Reply:
And on the line 62 ^

Reply:
this code is above and should work well to override props if present
```js
const {component: Root, props: rootProps} = getOverrideObject(
      overrides.Root,
      StyledRoot,
    );
    const {component: Action, props: actionProps} = getOverrideObject(
      overrides.Action,
      StyledAction,
    );
```